packages/components/src/Filter/FilterButton/FilterButton/FilterButton.module.css

```diff
@@ -0,0 +1,28 @@
+.filterButton {
+  gap: var(--spacing-xs);
+  justify-content: space-between;
+}
+
+/* keep max-width value in sync with other places using this container */
+@container filter-bar-wrapper (max-width: 767px) {
+  .filterButton {
+    flex-grow: 1;
+  }
+}
+
+.labelContainer {
+  display: inline-flex;
+  flex-wrap: wrap;
+}
+
+.hasSelectedValues {
+  font-weight: bold;
+}
+
+.labelSeparator {
+  margin-inline-end: var(--spacing-xs);
+
+  [dir="ltr"] [dir="rtl"] & {
+    margin-right: 0;
+  }
```

Comment:
This looks like a mistake since we now use logical properties 🤔 I think it should be safe to remove 🤞🏻